src/components/ItemDetailPage/ItemDetailPage.tsx

```diff
@@ -63,7 +63,7 @@ export default class ItemDetailPage extends React.PureComponent<Props> {
       <>
         <Section>
           <Row>
-            <Back absolute onClick={() => onNavigate(locations.collections())} />
+            <Back onClick={() => onNavigate(locations.collections())} />
```

Comment:
I think this could be turn into
```
<Back onClick={() => onNavigate(locations.collectionDetail(item.collectionId))} />
```
as well. I might make more sense to send the user to the collection detail

Reply:
@nicosantangelo we updated it to redirect to the detail page after talking with @LautaroPetaccio. Are you ok with that? :)

Reply:
sure! let me review the PR then :)